### ISSUE TITLE
AttributeError: 'list' object has no attribute 'encode'

### ISSUE THREAD

**Issue by Animakozak** [Labelbox/Labelbox#66]
So running JSON to COCO script lead me to this:

`Traceback (most recent call last):
  File "C:\Users\sined\AppData\Local\JetBrains\Toolbox\apps\PyCharm-P\ch-0\181.4668.75\helpers\pydev\pydevd.py", line 1664, in <module>
    main()
  File "C:\Users\sined\AppData\Local\JetBrains\Toolbox\apps\PyCharm-P\ch-0\181.4668.75\helpers\pydev\pydevd.py", line 1658, in main
    globals = debugger.run(setup['file'], None, None, is_module)
  File "C:\Users\sined\AppData\Local\JetBrains\Toolbox\apps\PyCharm-P\ch-0\181.4668.75\helpers\pydev\pydevd.py", line 1068, in run
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "C:\Users\sined\AppData\Local\JetBrains\Toolbox\apps\PyCharm-P\ch-0\181.4668.75\helpers\pydev\_pydev_imps\_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "C:/Users/sined/jsontococo/jsontococo.py", line 11, in <module>
    lb2co.from_json(labeled_data=labeled_data, coco_output=coco_output)
  File "C:\Users\sined\jsontococo\labelbox2coco\__init__.py", line 79, in from_json
    multipolygon = wkt.loads(data['Label'][cat])
  File "C:\Users\sined\jsontococo\venv\lib\site-packages\shapely\wkt.py", line 10, in loads
    return geos.WKTReader(geos.lgeos).read(data)
  File "C:\Users\sined\jsontococo\venv\lib\site-packages\shapely\geos.py", line 240, in read
    text = text.encode('ascii')
AttributeError: 'list' object has no attribute 'encode'`

The value it crashes on is:
`<class 'list'>: [{'x': 90, 'y': 319}, {'x': 526, 'y': 235}, {'x': 541, 'y': 145}, {'x': 83, 'y': 182}]`

**Comment by themaincharacter** [comment]
@Animakozak Were you able to resolve this issue? I am seeing the same thing error coming from the shapely/geos.py line 250.

**Comment by themaincharacter** [comment]
Turned out in the LabelFormat I needed to export this as WKT instead of raw XY coordinates.

**Comment by danielrasmuson** [comment]
Thanks for posting your finding @themaincharacter. If anyone hits an issue please post here and I'll reopen this issue.

**Comment by GottliebFreudenreich** [comment]
```
/Labelbox/scripts/labelbox2coco/__init__.py", line 79, in from_json
    multipolygon = wkt.loads(data['Label'][cat])
  File "/usr/local/lib/python3.5/dist-packages/shapely/wkt.py", line 10, in loads
    return geos.WKTReader(geos.lgeos).read(data)
  File "/usr/local/lib/python3.5/dist-packages/shapely/geos.py", line 250, in read
    text = text.encode('ascii')
AttributeError: 'list' object has no attribute 'encode'
```
I get the same error even when exporting as WKT

**Comment by GottliebFreudenreich** [comment]
Please reopen this issue. I've been trying every format and the scripts for JSON to COCO or JSON to PASCAL do not work! Am I missing something?
```


labelbox2coco/__init__.py:79: in from_json
    multipolygon = wkt.loads(data['Label'][cat])
/usr/local/lib/python3.5/dist-packages/shapely/wkt.py:10: in loads
    return geos.WKTReader(geos.lgeos).read(data)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <shapely.geos.WKTReader object at 0x7f735cf4dfd0>
text = [{'geometry': [{'x': 207, 'y': 208}, {'x': 212, 'y': 167}, {'x': 375, 'y': 168}, {'x': 378, 'y': 205}]}]

    def read(self, text):
        """Returns geometry from WKT"""
        if sys.version_info[0] >= 3:
>           text = text.encode('ascii')
E           AttributeError: 'list' object has no attribute 'encode'

/usr/local/lib/python3.5/dist-packages/shapely/geos.py:250: AttributeError

```

**Comment by feynmanliang** [comment]
Confirmed we can reproduce this, thanks for the report!

**Comment by feynmanliang** [comment]
I've pushed a fix for the pascal export in #110, and the COCO work is being tracked under #109 . I'm going to close this thread for now so we can centralize discussion under #109 .
